### DISCUSSION TITLE
SpiralLinux Social Media?

### DISCUSSION THREAD

**Discussion by Tagby76** [SpiralLinux/SpiralLinux-project#236] [General]
Hello,

I just wanted to know if the project has a Matrix, Discord, or SimpleX Chat channel?

**Comment by geckolinux** [comment]
Hi there,

I wouldn't call Matrix, Discord, or SimpleX "social media". But there is a Matrix channel at `#SpiralLinux:matrix.org` , mainly created to avoid squatters.

**Reply by Tagby76** [reply]
Yeah squatters are annoying. Do you wish the matrix chat were more active? Is that a particular interest to you?
